src/index.html

```diff
@@ -9,6 +9,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img src="./images/logo.png" alt="moyo-logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active nav__link" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
```

Comment:
as those are on the same level of nesting and there are quite a lot of them - you can add some empty lines between them for better readability
```suggestion
          <li class="nav__item">
            <a class="is-active nav__link" href="#apple">Apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#samsung">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#smartphones">Smartphones</a>
          </li>
          
          <li class="nav__item">
```